addons/assignGear/functions/fnc_setBoxContentsFromConfig.sqf

```diff
@@ -0,0 +1,34 @@
+#include "script_component.hpp"
+/*
+ * Author: Bailey
```

Comment:
I don't think I'm the author of this?

Reply:
[This is your code](https://github.com/BourbonWarfare/POTATO/blob/b1bf31beac2bc598f6fe0c6847a603612b908a38/addons/assignGear/functions/fnc_assignGearSupplyBox.sqf#L44), I just moved it to a function. If you want, I can throw my name on there too, but I didn't really write any of it, I just checked your work

Reply:
Oh yep, I thought it was for vehicles, not boxes

Reply:
Wait if this is and the other isn't, what's the difference in the code I'm missing? I think I changed the variable name to `_theContainer` in the other.

Reply:
I'm just going off of the blame. This is indeed not mine, viewing the history and blame pabst was the original author of the code path in `assignGearVehicle` which calls this. I'm just being a stickler about credits. Strictly speaking this should be credited to pabst as well, I must have just fucked up when I made my changes

Reply:
Gotcha, at that point then this function is also just me (didn't notice I cut out the part I had originally credited you for)